homeassistant/components/meteoalarm/binary_sensor.py

```diff
@@ -0,0 +1,100 @@
+"""Sensor for MeteoAlarm.eu."""
```

Comment:
Please add "binary" to all comments that mention "sensor".

Reply:
Also fixes this, thanks for mention that.